build/msw-vc-2017/copy_files.bat

```diff
@@ -6,5 +6,5 @@ copy /Y ..\..\3rd\sorttable.js\sorttable.js %1res
 IF NOT EXIST %1po mkdir %1po
 xcopy /Y /E /I ..\..\po\*.mo %1po\en
 copy /Y ..\..\doc\contrib.txt %1
-IF NOT EXIST %1help xcopy /Y /E /I ..\..\doc\help\*.* %1help
+IF NOT EXIST %1help xcopy /Y /E /I ..\..\docs %1help
```

Comment:
According to my [proposal](https://github.com/moneymanagerex/moneymanagerex/pull/1136/files#diff-af3b638bc2a3e6c650974192a53c7291R271) to rename dir `docs` to `help`, this line needs to be altered too.